src/index.css

```diff
@@ -48,6 +57,15 @@ code {
   align-items: center;
 }
 
+.Add-Book-btn {
```

Comment:
Though your design is not bad, this section doesn't look like the one in the design. In order for your design to be perfect please fix your button borders and the color of your default select, you should try to match the template as closely as possible which means following the guidelines for borders and color in your elements.

Please try adding the corresponding CSS property to stick to the design as much as possible. For example, why don't you try removing the borders to the button and the color of your select option for this section?

Check the image below to have a better understanding.

![Capture](https://user-images.githubusercontent.com/75709904/146547731-c1c04f1c-6561-4b1a-9401-f94892c9ac78.PNG)

Reply:
Thanks, Implemented edits